### ISSUE TITLE
Workflow B: ARAX not sending >100 input CURIES during expansion

### ISSUE THREAD

**Issue by karafecho** [NCATSTranslator/minihackathons#317]
This issue is to report that ARAX is not calling KPs when there are more than 100 input CURIEs in the expansion.

**Comment by dkoslicki** [comment]
Yes, that is by design. We have found that most KPs are not performant enough to handle such queries within the default time-out. So instead of waiting for them to time out, we do not decide to even attempt. @amykglen may have more insight

**Comment by karafecho** [comment]
Yes, I understand that the restriction is by design. I'm wondering, however, if you might be able to suggest a workaround that would allow ARAX to pull in additional KPs, or is this strictly a KP issue?

**Comment by dkoslicki** [comment]
I think it's mainly a KP performance issue, but @amykglen may be able to shed more light on possible workarounds.

**Comment by amykglen** [comment]
yes, we're going to discuss this at our team meeting tomorrow - will post back here after that.

**Comment by karafecho** [comment]
Great! Thanks, David and Amy!

**Comment by amykglen** [comment]
ok, ARAX now sends queries to KPs even if they have >100 input curies. we're also revisiting our default timeouts and expect to increase those within the next week!

**Comment by karafecho** [comment]
This is great! Thanks, Amy.
